ktlint-ruleset-standard/src/main/kotlin/com/github/shyiko/ktlint/ruleset/standard/SpacingAroundParensRule.kt

```diff
@@ -23,7 +29,11 @@ class SpacingAroundParensRule : Rule("paren-spacing") {
             val nextLeaf = node.nextLeaf()
             val spacingBefore = if (node.elementType == LPAR) {
                 prevLeaf is PsiWhiteSpace && !prevLeaf.textContains('\n') &&
-                    prevLeaf.prevLeaf()?.elementType == IDENTIFIER && (
+                    (
```

Comment:
This formatting looks pretty wacky to me, but AS seems to agree it's correct.

Reply:
It's weird the this open paren is on a new line but the open paren on line 36 is not.

Reply:
Yeah I agree. There's definitely bugs with the indent formatting rule but I can't always figure them out.